TrafficSimulator/Assets/Utilities/CarSpawner.cs

```diff
@@ -164,16 +165,10 @@ private void SpawnCars()
 
                     _carCounter++;
 
-                    SetOffset(_offset);
+                    _offset += _lanes[i].Length / carsToSpawn;
```

Comment:
Protect against division by zero

Reply:
Already protected against division by zero because of previous if-statement